health_check_storage/base.py

```diff
@@ -45,28 +45,36 @@ def get_file_name(self):
     def get_file_content(self):
         return b'this is the healthtest file content'
 
+    def check_save(self, file_name, file_content):
+        storage = self.get_storage()
+        # save the file
+        file_name = storage.save(
+            file_name, ContentFile(content=file_content)
+        )
+        # read the file and compare
+        if not storage.exists(file_name):
+            logger.exception('File does not exist')
```

Comment:
I think it might be a good idea to start translating those messages. How about we start using `gettext` as a start?